### DISCUSSION TITLE
Allow loading of some Locomotion objects / Linking Locomotion

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#18164] [Feature requests]
I believe I forgot to make a FR for this when I was researching this a few months back

I think it'd be nice if we could load some dat files from locomotion, maybe even similar to RCT1 to link Locomotion and it'd give you access to some extra things.
stuff like rock1.dat and rock2.dat which are surfaces and edges combined in one dat file.
And stuff like the soundtrack

**Comment by IntelOrca** [comment]
The music should be simple.
Some of the sounds should be simple.
Anything in a Loco object requires knowledge of each loco object type. Unfortunately there is no easy way to skip to the image table of an object without knowing exactly how many bytes to read from the header and variable string table. So each loco object type header size would need to be known for this to work.
The JSON objects would just do `$LOCO:OBJDATA/ABC.DAT[..]` or `$LOCO:DATA/CSS1.DAT`.
